### ISSUE TITLE
 Review Design Wireframe: Meet the children page

### ISSUE THREAD

**Issue by lem31** [NoroffFEU/musikkforandrerliv.no#93]
- [x] Visual Hierarchy
- [x] Layout
- [x] Consistency
- [x] Screen size
- [x] Usability Principles
- [x] Web Content Accessibility Guidelines (WCAG)
- [x] Colors
- [x] Fonts
- [x] Whitespace and padding
- [x] Image quality
- [x] Iconography
- [x] Micro interactions
- [x] Error states and feedback
- [x] Performance Optimization
- [x] Design documentation

See Design wireframe/prototype checklist for more info

**Comment by Vickabu** [comment]
Arrows is connected to "more" not "about us" its to indicate a drope down if more content is added. 

*Since footer and nav is mention i several issues, but with different comments - it should have been one issue with one precise description. Same with the newsletter. and the arrows *

**Comment by aggicreative555** [comment]
1. Buttons 
- [ ] Accessibility (Montserrat Font, 24px, semi-bold)
- [ ] Padding: 16px 24px 16px 24px 

![Image](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/142678571/e7f5f1b8-edf7-4fb0-9d54-ab4f27a0c94f)


2. Newsletter 

- [ ] Distinguish from the rest of the page
- [ ] Make Mobile & Desktop text consistent
- [ ] Accessibility (18px font, black) 
- [ ] Padding (Desktop - 80px top/bottom, Mobile 32px top/bottom)


![Image](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/142678571/4c805f14-f638-4d89-8a25-34868d133344)


Context:

![Image](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/142678571/3fb78ee8-f6e8-4404-866c-e724b939ab71)


Component:

![Image](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/142678571/cfda16d4-9233-45fd-8ce2-cc16166ac1ae)


Ask anything :)

**Comment by Vickabu** [comment]
This pages is now scrapped from the client. Thiss issue can therefore be closed - other thing that dont refere to the actull page is mentioned in other issues as well.
